cloud-formation/cloud-formation.yaml

```diff
@@ -169,12 +187,12 @@ Resources:
   EventbriteImagesELBDNSrecord:
     Type: 'AWS::Route53::RecordSet'
     Properties:
-      HostedZoneId: /hostedzone/Z1E4V12LQGXFEC
+      HostedZoneId: /hostedzone/Z8SGOO5042FR9
       Comment: CNAME for AWS ELB
       Name: !Join
```

Comment:
Could just build the whole name using Sub, for clarity?

Reply:
Updated